src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__content {
+  background: url(./images/star.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__content:last-child {
+  margin: 0;
+}
+
+.stars--1 div:nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 div:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 div:nth-child(-n+1),
.stars--2 div:nth-child(-n+2) {
  background-image: url(./images/star-active.svg);
}
```
об'єднай всі через кому